### PR TITLE
randomise dispute calculation

### LINKED ISSUE #373: randomise dispute calculation
# Rationale
Every voter looks for dispute from 0-n.
We need to randomise this order.

### PR DESCRIPTION
**Pull request by shekhar2807**
# Description

Used a inbuilt function of random library rand.Perm(n) which returns the random permutation from  0 to n-1.

Fixes #373

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Contracts version:
* Hardware:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by ashish10677** [comment]
@shekhar2807 there are conflicts.

**Comment by shekhar2807** [comment]
> @shekhar2807 there are conflicts.

@ashish10677 yes looking into it.

File: cmd/dispute.go [hunk 1]
```diff
@@ -33,8 +34,10 @@ func (utilsStruct UtilsStruct) HandleDispute(client *ethclient.Client, config ty
 	log.Debug("Locally calculated data:")
 	log.Debugf("Medians: %d", medians)
 
-	for i := 0; i < len(sortedProposedBlockIds); i++ {
-		blockId := sortedProposedBlockIds[i]
+	randomSortedProposedBlockIds := rand.Perm(len(sortedProposedBlockIds)) //returns random permutation of integers from 0 to n-1
+
+	for i := 0; i < len(randomSortedProposedBlockIds); i++ {
```
Comment (ashish10677) [inline cmd/dispute.go:39]: You can try the `for...range` loop here. Might simplify the code.

**Review by ashish10677 — CHANGES_REQUESTED** [review]

**Review by ashish10677 — APPROVED** [review]
LGTM

**Pull request merged**